plugins/rails/rails.plugin.zsh

```diff
@@ -62,6 +62,17 @@ alias rrg='rake routes | grep'
 alias rt='rake test'
 alias rmd='rake middleware'
 alias rsts='rake stats'
+alias rtasks='rake --tasks'
+alias ra='rake about'
+alias rdv='rake db:version'
+alias rtcc='rake tmp:cache:clear'
+alias rtssc='rake tmp:sessions:clear'
+alias rtsc='rake tmp:sockets:clear'
+alias rtc='rake tmp:clear'
+alias rtmp='rake tmp:create'
+alias rst='rake secret'
```

Comment:
`rst` alias is duplicated with https://github.com/robbyrussell/oh-my-zsh/blob/master/plugins/repo/repo.plugin.zsh#L17